eZ/Publish/Core/REST/Server/Controller/Content.php

```diff
@@ -473,10 +471,8 @@ public function updateVersion($contentId, $versionNumber, Request $request)
 
         try {
             $this->repository->getContentService()->updateContent($versionInfo, $contentUpdateStruct);
-        } catch (ContentValidationException $e) {
-            throw new BadRequestException($e->getMessage());
         } catch (ContentFieldValidationException $e) {
-            throw new BadRequestException($e->getMessage());
+            throw new RESTContentFieldValidationException($e->getFieldErrors(), $e->getMessage());
```

Comment:
hmm, maybe ignore, I see orignal exception is probably passed in as $data to on visit()